components/table/TableRow.js

```diff
@@ -60,6 +60,9 @@ const factory = (Checkbox) => {
     renderInput (key, value) {
       const index = this.props.index;
       const inputType = utils.inputTypeForPrototype(this.props.model[key].type);
+      if (this.props.model[key].render) {
+        return this.props.model[key].render(this.props.data);
```

Comment:
I don't get this. Is it like a method to notify that rendered had happened? If so, naming it `onRender` would me more appropriated IMO